django_react_proj/users-fe/src/App.js

```diff
@@ -1,12 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Card, Menu } from 'antd';
 import Header from './components/Header/Header';
 import Home from './components/Home/Home';
+import Subtract from './components/SummingTime/SummingTime';
 
-const App = () => (
-  <>
-    <Header />
-    <Home />
-  </>
-);
+const App = () => {
+  const [noTitleKey, setState] = useState('1');
```

Comment:
setting function name should be similar to state name. 'setState' is not a good option...

Reply:
Changed